### ISSUE TITLE
Add to Citadel, Umbrel, RaspiBlitz app store 

### ISSUE THREAD

**Issue by bumi** [getAlby/lndhub.go#174]
Those nodes OS have appstore. It would be nice to be able to install lndhub there from within the app store. 
Citadel and Umbrel use Docker, so that should be fairly easy I think. And we can use sqlite by default, so no additional dependency is needed.

**Comment by prusnak** [comment]
> And we can use sqlite by default, so no additional dependency is needed.

This has changed when https://github.com/getAlby/lndhub.go/pull/199 was merged, but still shouldn't be a blocker adding lndhub.go to Citadel, Umbrel, RaspiBlitz, etc.

**Comment by bumi** [comment]
yeah, though I'd like to bring SQLite support back. it should be easy enough and I think only the migrations need to be adjusted.

**Comment by AaronDewes** [comment]
LNDHub.go is now in alpha for the Citadel app store and is scheduled to be released (with Postgres) as part of Citadel 0.1.0.

**Comment by bumi** [comment]
whooot! yay, nice!

**Comment by prusnak** [comment]
Nix-Bitcoin (draft) PR for including LndHub.go is here: https://github.com/fort-nix/nix-bitcoin/pull/519

**Comment by bota87** [comment]
> yeah, though I'd like to bring SQLite support back.

Yes please, it's a lot easier to configure and backup for a low usage setup like my
